app/next-client-app/components/scanreports/ScanReportTableUpdateForm.tsx

```diff
@@ -103,6 +121,78 @@ export function ScanReportTableUpdateForm({
               />
             </div>
 
+            <div className="flex gap-2 mt-2">
+              <h3 className="flex">
+                This table contains data mapped to the DEATH table in OMOP CDM?
+                <Tooltips content="If 'YES', concepts added here with domains RACE, ETHNICITY and GENDER will be mapped to the PERSON table. ALL of other concepts added here will be mapped to the DEATH table" />
+              </h3>
+              <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+                <DialogTrigger
+                  asChild
+                  className={cn(
+                    "bg-carrot-200",
+                    values.death_table && "bg-carrot"
+                  )}
+                >
+                  <Switch
+                    checked={values.death_table}
+                    onCheckedChange={(checked) => {
+                      setPendingValue(checked);
+                      setIsDialogOpen(true);
+                    }}
+                    disabled={!canUpdate || scanreportTable.death_table}
+                  />
+                </DialogTrigger>
+                <DialogContent>
+                  <DialogHeader>
+                    <DialogTitle>Let's double check</DialogTitle>
```

Comment:
Model improvement suggestion

## Please Confirm Your Choice

Are you sure you want to set this table as a DEATH table? Doing so will result in the following:

- Concepts added to this table with domains **RACE**, **ETHNICITY**, and **GENDER** to be mapped to the **PERSON** table in OMOP CDM
- All other domain concepts will be mapped to the **DEATH** table in the OMOP CDM.

⚠️ Warning: Once saved, this setting is permanent and cannot be undone.